### PR TITLE
[DOCS] Adding MO Python API `convert_model()` commands

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Implementation of changes proposed in a draft PR: https://github.com/openvinotoolkit/openvino/pull/16761

### CODE REVIEW

**Comment by popovaan** [comment]
It was discovered that many tutorial from documentation have stopped to work and some of them need removing.
So we need to remove part of changes and check all the remaining commands before merging.

@sgolebiewski-intel please remove changes from all files except following files:
Deep_Learning_Model_Optimizer_DevGuide.md
FP16_Compression.md
Additional_Optimizations.md
troubleshooting-steps.md

And please check all commands in these files.

File: docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md [hunk 1]
```diff
@@ -40,9 +40,21 @@ How to Run Model Optimizer
...
+
+       .. code-block:: sh
+
+          mo --input_model INPUT_MODEL
+
+    .. tab-item:: MO Python API
+       :sync: mo-python-api
+
+       .. code-block:: sh
+
```
Comment (popovaan) [inline docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md:56]: Please add following import to all MO Python API examples:

`from openvino.tools.mo import convert_model`

File: docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md [hunk 2]
```diff
@@ -40,9 +40,21 @@ How to Run Model Optimizer
...
+       :sync: cli-tool
+
+       .. code-block:: sh
+
+          mo --input_model INPUT_MODEL
+
+    .. tab-item:: MO Python API
+       :sync: mo-python-api
+
+       .. code-block:: sh
```
Comment (popovaan) [inline docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md:55]: Why `code-block:: sh` is used? Can we use `code-block:: python`?

File: docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md [hunk 3]
```diff
@@ -74,24 +100,64 @@ Below is a list of separate examples for different frameworks and Model Optimize
...
+   
+             mo --saved_model_dir BERT --input mask,word_ids,type_ids --input_shape [2,30],[2,30],[2,30]
+   
+       .. tab-item:: MO Python API
+          :sync: mo-python-api
+   
+          .. code-block:: python
+   
+             from openvino.tools.mo import convert_model
+             ov_model = convert_model(saved_model_dir="BERT", input=["mask", "word_ids", "type_ids"], input_shape=[[2,30],[2,30],[2,30]])
```
Comment (popovaan) [inline docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md:138]: @sgolebiewski-intel did you check these commands? 
Bert model is expected to not work with batch **2**. So most likely shapes here need to be changed.

Comment (sgolebiewski-intel) [reply docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md:138]: Yes, confronted it with what you have proposed in :

https://github.com/openvinotoolkit/openvino/pull/16761/files#diff-6a3db6969c80ae509adac48b7a6b3523fb542f7f74ee33d2fc95fa8f999ef909

:)

Comment (popovaan) [reply docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md:138]: Yes, I wrote the identical python API command to original CLI command. But I didn't check that original command still works. We observed that some commands stopped to work, so we need to check them. 

Can you please take a bert model and check that command from this page work?
If you will have troubles with finding the model contact me or @rkazants

File: docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md [hunk 4]
```diff
@@ -74,24 +100,64 @@ Below is a list of separate examples for different frameworks and Model Optimize
...
-
-      For more information, refer to the :doc:`Converting a TensorFlow Model <openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow>` guide.
+   .. tab-set::
+   
+       .. tab-item:: MO command line tool
+          :sync: cli-tool
+   
+          .. code-block:: sh
+   
+             mo --saved_model_dir BERT --input mask,word_ids,type_ids --input_shape [2,30],[2,30],[2,30]
```
Comment (popovaan) [inline docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md:130]:
```suggestion
             mo --saved_model_dir BERT --input_shape [2,30],[2,30],[2,30]
```

File: docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md (hunk 3 above)
Comment (popovaan) [inline docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md:138]:
```suggestion
             ov_model = convert_model("BERT", input_shape=[[2,30],[2,30],[2,30]])
```

**Review by popovaan — CHANGES_REQUESTED** [review]

**Comment by sgolebiewski-intel** [comment]
The PR can be closed, as long as proposed changes have been propagated to https://github.com/openvinotoolkit/openvino/pull/17454 and further work with 'MO documentation reorganization'  implemented there.

**Pull request closed without merging**